examples/basic/main.go

```diff
@@ -8,16 +8,18 @@ import (
 	"os"
 	"time"
 
+	msgs "github.com/messenger"
```

Comment:
Travis is breaking because this repository doesn't exist, should probably be changed to: github.com/paked/messenger

I also don't think using an alias on this package is a good idea. It's fine for your own code, but should be avoided because this is an "official" example.

Reply:
Yeah... I am really sorry about that. I developed and it worked for me locally so I didn't bother to check over. I am so sorry! :upside_down_face: I didn't realize it'd reference my second commit as I wasn't really ready to submit the PR.

So I should make a fix for those then.